index.html

```diff
@@ -199,7 +199,21 @@ <h2 class="subt">Skills</h2>
           </div>
         </div>
       </section>
-    </main>
-    <footer></footer>
+      <footer class="contact-form">
+        <section class="form-section">
+          <div class="contact-me">
+            <h2 >Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <form action="send" class="form">
```

Comment:
It's important that you use [Formspree service](https://formspree.io/html) in order to get all of the data being inputted on the form 🤓 Please add this functionality to your form so that we meet the requirements 📝

Reply:
Done!